src/inc/eventtracebase.h

```diff
@@ -531,6 +531,7 @@ namespace ETW
                 ManifestModule=0x8,
                 IbcOptimized=0x10,
                 ReadyToRunModule=0x20,
+                PartialReadyToRunModule=0x40,
```

Comment:
@vancem Should this flag be listed in ClrEtwAll.man as well?

Looks like a few of the recently added flags are missing in ClrEtwAll.man.

Reply:
Is this blocking or can we just file an issue to look what we need to do for values `> 0x8` and merge this?

Reply:
It is ok to file an issue.

Reply:
Thanks! Filed #22790 in the Tracing area path for 3.0.

Reply:
> @vancem Should this flag be listed in ClrEtwAll.man as well?

The answer is yes, I will update the issue with more.

I have already updated PerfVIew's enum for this so you will see it parsed in the 'events' view in PerfView